internal/config/flags.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"flag"
+	"os"
+)
+
+const (
+	defaultSrvAddr = "localhost:8080"
+	defaultResAddr = "http://localhost:8080"
+)
+
+var (
+	ServerAddr string
+	ResultAddr string
+)
+
+var (
+	srvFlagAddr string
+	resFlagAddr string
+)
+
+func ParseFlags() {
+	srvEnvAddr, srvEnvExist := os.LookupEnv("SERVER_ADDRESS")
```

Comment:
В Go принято проверять условия более лаконично:

```suggestion
	srvEnvAddr, ok := os.LookupEnv("SERVER_ADDRESS")
```

Reply:
@s-shpak  как быть с 
```
resEnvAddr, resEnvExist := os.LookupEnv("BASE_URL")
```
ведь нужно проверить оба флага, тут можно оставить resEnvExist ?

Reply:
Делай проверку `ok` сразу после вызова `os.LookupEnv`. Вообще, хорошей практикой считается групировать логику близко, если это возможно